### DISCUSSION TITLE
Ability to hide the endpoint

### DISCUSSION THREAD

**Discussion by dingzhou6** [scalar/scalar#1728] [Ideas]
It would be nice to have the ability to hide endpoints that are currently under testing or intended for internal use only.

**Comment by hanspagel** [comment]
Great idea!

**Reply by hanspagel** [reply]
Submitted a PR :)
https://github.com/scalar/scalar/pull/1729

**Reply by dingzhou6** [reply]
sorry, the `x-internal` seems like not working , it never hide the endpoint in the spec

**Reply by itizarsa** [reply]
@dingzhou6 OpenAPI has a property hidden. If you use that, the generated spec won't list the endpoint. We use that to hide internal endpoints
